lib/redis/commands/sorted_sets.rb

```diff
@@ -722,6 +722,18 @@ def zinterstore(*args)
       end
       ruby2_keywords(:zinterstore) if respond_to?(:ruby2_keywords, true)
 
+      # This command is similar to ZINTER, but instead of returning the result set,
+      #   it returns just the cardinality of the result.
+      #
+      # @param [Array<String>] keys list of sorted sets
+      # @param [Integer] limit will short circuit operation if limit is met
```

Comment:
Requires definition of return for YARDoc.

relates in https://github.com/redis/redis-rb/pull/1201/files#r1315881674

```suggestion
      # @return [Integer] cardinality of the intersection
```